.rubocop.yml

```diff
@@ -51,3 +51,5 @@ Style/HashTransformKeys:
   Enabled: false
 Style/HashTransformValues:
   Enabled: false
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
Please don't modify the `.rubocop.yml` otherwise your projects will not be approved.